scripts/.style.yapf

```diff
@@ -0,0 +1,17 @@
+[style]
+based_on_style = pep8
+blank_lines_between_top_level_imports_and_variables=2
+blank_line_before_class_docstring=True
+blank_line_before_module_docstring=True
+blank_line_before_nested_class_or_def=True
+coalesce_brackets=True
+column_limit=90
+continuation_indent_width = 2
+dedent_closing_brackets=True
+indent_dictionary_value=True
+indent_width = 2
```

Comment:
4 please -- personal preference :)

Reply:
same for continuation indent width

Reply:
😬 😲 I'm big on 2 personally.....any other python devs we could bring in as a tiebreaker?

Reply:
Given that I've done 99% of the work in this project and likely will continue to do most of it I'm going to hardline on optimizing for myself and the style I use in all my other projects. We can rediscuss and/or change if I do end up handing over the project or ending up with a more even split of workload.

Reply:
Fair point, works for me

Reply:
Done